### PR TITLE
Add Cog file to build Docker image

### PR DESCRIPTION
**Pull request by jxmorris12**
Hey @kongqiuqiang ! 👋 

I've been working with this model for awhile since it's one of the best open-source models for piano transcription. I recently started working with [Replicate](https://replicate.com/) to make machine learning models more reproducible. I wanted to make a cool demo for piano transcription, which ended up as [synthviz](https://github.com/jxmorris12/synthviz), a small Python package for making Synthesia-esque videos from piano transcription. I wrapped your model and my visualization up in a Docker environment (which is also available to try as a demo on Replicate). 

This pull request makes it possible to run your model inside a Docker environment, which makes it easier for other people to run it.  We're using an open source tool called [Cog](https://github.com/replicate/cog) to make this process easier.

This also means we can make a web page where other people can try out your model! View it here: https://replicate.com/bytedance/piano-transcription

[Claim your page here](https://replicate.com/bytedance) so you can edit it, and we'll feature it on our website and tweet about it too.

### CODE REVIEW

**Comment by fivestones** [comment]
Now that this is merged, I have tried this out on replicate. However, it's only output that I can see is the video file with the midi playing and the piano keys visualization. How can I get the docker image from replicate (or from anywhere) to use to install this software on my own machine?

tldr; Where is the docker image?

Thanks!

**Comment by jxmorris12** [comment]
@bfirsh @andreasjansson can you provide some more info on how to obtain & use the docker image?

**Comment by bfirsh** [comment]
You can click the Docker tab here: https://replicate.com/bytedance/piano-transcription/api

You can also do `cog run` locally in this repository to build and run the Docker image.

**Comment by qiuqiangkong** [comment]
Hi Jack,

    Thanks for the effect!

Best wishes,
Qiuqiang

**Comment by fivestones** [comment]
> You can click the Docker tab here: https://replicate.com/bytedance/piano-transcription/api
> 
> You can also do `cog run` locally in this repository to build and run the Docker image.

Thanks! Great.
